feature/movie/src/main/java/com/chan/movie/ui/main/MovieSearchViewModel.kt

```diff
@@ -44,16 +44,34 @@ class MovieSearchViewModel @Inject constructor(
     private val saveList = mutableListOf<ItemDto>()
 
     private val pagingInfo = PageInfo(PageData())
-    private var job: Job? = null
+
+    private val _searchQuery = MutableStateFlow("")
+    fun setSearchQuery(query: String) {
+        _searchQuery.value = query
+    }
```

Comment:
```suggestion
    val searchQuery: StateFlow<String> get() = _searchQuery.asStateFlow()
```
P2. public getter를 활용하시면 좋을 것 같아요

Reply:
67dd5cf656b6fc335dc675cbba99a59295720638 debunce에 대한 처리를 뷰로 이관하면서 삭제하였습니다.